packages/tv/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@my-app/tv",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "start": "react-native start",
+    "android": "emulator -avd Android_TV; react-native run-android",
+    "ios": "react-native run-ios --simulator 'Apple TV' --scheme 'tv-tvOS'",
+    "studio": "studio android",
+    "xcode": "xed ios",
+    "pods": "pod-install",
+    "test": "jest",
+    "lint": "eslint . --ext .js,.jsx,.ts,.tsx"
+  },
+  "dependencies": {
+    "@my-app/app": "*",
+    "@react-native-async-storage/async-storage": "^1.15.7",
+    "react": "17.0.1",
+    "react-native": "npm:react-native-tvos@0.64.2-2"
```

Comment:
TIL it's on a separate package! Thanks 🙌

Reply:
Yeah I was surprised too!